tower-scripts/bin/cicd-control.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash -e
-
 START_DIR=$(dirname "${BASH_SOURCE[0]}")
+
 GIT_ROOT="/home/opsmedic/aos-cd/git"
 TMPDIR="$HOME/aos-cd/tmp"
 mkdir -p "${TMPDIR}"
 
-VALID_ARGUMENTS=(cicd_docker_version cicd_openshift_ansible_build cicd_openshift_version)
+VALID_ARGUMENTS=(cicd_docker_version cicd_openshift_ansible_build cicd_openshift_version cicd_yum_main_url cicd_yum_openshift_ansible_url)
```

Comment:
I wasn't expecting to see `cicd_yum_main_url` here. Maybe it will make sense with the other PR?

Reply:
Yep, make sense now.